src/libsync/abstractnetworkjob.cpp

```diff
@@ -144,19 +150,26 @@ bool AbstractNetworkJob::needsRetry() const
 }
 
 
-void AbstractNetworkJob::sendRequest(const QByteArray &verb, const QUrl &url,
+void AbstractNetworkJob::sendRequest(const QByteArray &verb,
     const QNetworkRequest &req, QIODevice *requestBody)
 {
     _verb = verb;
     _request = req;
-    _request.setUrl(url);
     _requestBody = requestBody;
+    // TODO:
+    if (_request.url().isEmpty()) {
```

Comment:
One possibility for this to happen is https://github.com/owncloud/client/pull/9449/files#diff-1b4697b262724707f8fa1a7b3fa265141cc0e97d67dc0a2a38975ebcd1ad9612R697

Not sure how to handle this properly.

Reply:
done